infra/gcp/deploy-cip-auditor.sh

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env bash
+#
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO(listx): CONVERT THIS TO TERRAFORM!
+
+# Deploy Cloud Run service named "cip-auditor".
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+CLOUD_RUN_SERVICE_NAME="cip-auditor"
+SUBSCRIPTION_NAME="cip-auditor-invoker"
+SERVICE_ACCOUNT="k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com"
```

Comment:
This SA has write permissions.  Should we make one with less access for this audit task?

Reply:
That would be ideal, yes. Ack

Reply:
Do you want to add that to this PR?  Call empower_artifact_auditor() in ensure-prod-storage.sh